Content.Server/CharacterInfo/CharacterInfoSystem.cs

```diff
@@ -27,6 +27,7 @@ private void OnRequestCharacterInfoEvent(RequestCharacterInfoEvent msg, EntitySe
 
         var conditions = new Dictionary<string, List<ConditionInfo>>();
         var jobTitle = "No Profession";
+        var briefing = "No Briefing (And this should not be showing!)";
```

Comment:
localize

Reply:
This shouldn't show under normal conditions so I don't see why it has to be localized